VAMobile/src/components/Templates/CategoryLanding.tsx

```diff
@@ -27,9 +28,11 @@ export type CategoryLandingProps = {
   headerButton?: headerButton
   /** Optional ScrollView props to pass through to VAScrollView if desired */
   scrollViewProps?: VAScrollViewProps
+  /** Optional waygate to be made a requirement later */
+  waygate?: WaygateToggleType
```

Comment:
I'm making all the templates conditional to start and then required once I've rolled them out to all the screens. This will simplify the template code to remove the conditional.